source/_theme/pydy/layout.html

```diff
@@ -36,17 +36,25 @@
   <div class="main-nav" id="main-nav">
     <div id="site-desc" class="site-desc">
       <div class="logo">
-        <img class="image"src="_static/logo.svg">
+        <img class="image"src="/_static/logo.svg">
```

Comment:
Should we be using this: http://sphinx-doc.org/templating.html#logo in a Jinja template?